unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Video/Plugins/WebVideoPlayer.cs

```diff
@@ -155,12 +161,59 @@ public float GetDuration()
 
         public VideoState GetState()
         {
-            return (VideoState)plugin.GetState(videoPlayerId);
+            return plugin.GetState(videoPlayerId);
+        }
+
+        public void SetAsLoading()
+        {
+            UpdateTextureConservingAspectRatio(Resources.Load<Texture2D>(VIDEO_LOADING_PATH), true);
```

Comment:
We should be loading that resource only once and cache the result, this is also a good opportunity to use addressable and remove that texture from the Resources folder

Reply:
Addressables is the idea, it was commented also by Ashley, right now as I am forcing the error for QA, this PR should be ignored by devs ^^